modules/jaeger-deploy-default.adoc

```diff
@@ -1,8 +1,8 @@
 ////
-This PROCEDURE module included in the following assemblies:
+This module included in the following assemblies:
 - rhbjaeger-deploying.adoc
 ////
-
+:_content-type: PROCEDURE
```

Comment:
Refer to this guideline example: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Observation: I am seeing a line space before the content-type attribute in the guideline example. Please follow this for all the files (global comment).